modules/ROOT/Edits/create-inbound-message-flow.adoc

```diff
@@ -0,0 +1,119 @@
+
+=== Steps
+
+. From the Anypoint Partner Manager navigation menu, select *Message Flows*.
+. Click *New message flow > Receive from partner*.
+. Either select an existing partner from the list or select *New* to create a new partner, as described in xref:create-partner.adoc#[Create a partner].
++
+A new message flow is created.
++
+. <<receiving-endpoint,Configure the receiving endpoint>>.
+. <<source-message-type,Configure the source message type>>.
+. <<import-map,Import the translation map>>.
+. <<verify-message-flow,Verify that the message flow is complete>>.
+
+[receiving-endpoint]
+== Specify the Receiving Endpoint
+
+Specify the endpoint that will receive the message you are sending. You can do either of the following:
+
+* Select an existing endpoint owned by the host, target partner, or 3rd party connection.
++
+If you select an endpoint owned by a 3rd party connection, the inbound message goes through that connection.
++
+* Create a new receiving endpoint for this message flow.
+
+To specify the receiving endpoint:
+
+. In the message flow, expand the *Receiving from <partner>* section and click *Select endpoint*.
+. In the *Receiving endpoint* section, click *Select*.
+. In the *Select an endpoint: Receive from <partner>* window, do either of the following:
+* Select an existing receiving endpoint and click *Select*.
+* Click *New* to create a new endpoint, as described in xref:<<create-endpoint.adoc, Create and Configure Endpoints>>.
++
+The endpoint is added to the message flow, and a message appears indicating that the endpoint is missing an identifier.
+
+=== AS2 Sender
+
+If the protocol of the receive endpoint is AS2, select the AS2 Sender from the dropdown menu:
+
+* If your organization will receive the AS2 messages directly from the partner for whom the message flow is created, select the partner’s name (default).
+* If your organization will receive the AS2 messages for the partner via a 3rd party connection, such as a VAN or 3PL, select the name of the 3rd party connection. The AS2 identifier and certificate of the selected 3rd party connection will be used in AS2 message validation.
+* If you selected AS2 as the receive to partner endpoint protocol, you must import the AS2 certificate of the partner or the 3rd party connection sending the AS2 message, if a certificate already does not exist in the partner or 3rd party connection’s profile.
+
+[source-message-type]
+== Configure the Source Message Type
+
+. In the message flow, expand the *Source message* section and click *Select*.
+. In the *Select message type* window:
+.. To use an existing message type, select the message type from the list and then click *Select*.
+.. To create a new message type, click *New* next to the search box, configure the X12 or EDIFACT xref:partner-manager-create-message-type.adoc[message type], and click *Save*.
+The message type and its validation settings are added to the message flow.
```

Comment:
I find this similarly hard to parse; how about using two mutually exclusive "To..." steps:
```suggestion
. To use an existing message type, select the message type from the list in the *Select message type* window and then click *Select*.
. To create a new message type, click *New* next to the search box in the *Select message type* window, configure the X12 or EDIFACT xref:partner-manager-create-message-type.adoc[message type], and click *Save*.
The message type and its validation settings are added to the message flow.
```
...although I have to admit that I would prefer to see substeps for the second item, which comprises three actions....